starboard/shared/starboard/player/dmp_demuxer.h

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_SHARED_STARBOARD_PLAYER_DMP_DEMUXER_H_
+#define STARBOARD_SHARED_STARBOARD_PLAYER_DMP_DEMUXER_H_
+
+#include <map>
+#include <utility>
+
+#include "starboard/shared/starboard/player/video_dmp_reader.h"
+
+namespace starboard {
+namespace shared {
+namespace starboard {
+namespace player {
+namespace video_dmp {
+
+class DmpDemuxer {
```

Comment:
Consider adding some comments here describing what DmpDemuxer is.

Reply:
done

Reply:
Reopen